src/lib.mli

```diff
@@ -39,6 +39,7 @@ val main_module_name : t -> Module.Name.t option Or_exn.t
 val wrapped : t -> Wrapped.t option Or_exn.t
 
 val virtual_ : t -> Lib_modules.t Lib_info.Source.t option
+val implements : t -> t Or_exn.t option
```

Comment:
I would prefer a function like `is_impl` because we change the internals of lib.ml often enough that it makes sense to keep a minimal interface.

Of course, if we include the implementation in the index we will need a function of this form after all. But if not, let's stick to a more minimal interface.

Reply:
:+1: sounds like a good idea.